google-cloud-clients/google-cloud-language/samples/src/main/java/com/google/cloud/examples/language/v1/LanguageClassifyGcs.java

```diff
@@ -0,0 +1,99 @@
+
+package com.google.cloud.examples.language.v1;
+
+import com.google.cloud.language.v1.ClassificationCategory;
+import com.google.cloud.language.v1.ClassifyTextRequest;
+import com.google.cloud.language.v1.ClassifyTextResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+public class LanguageClassifyGcs {
+  // [START language_classify_gcs]
+  /*
+   * Please include the following imports to run this sample.
+   *
+   * import com.google.cloud.language.v1.ClassificationCategory;
+   * import com.google.cloud.language.v1.ClassifyTextRequest;
+   * import com.google.cloud.language.v1.ClassifyTextResponse;
+   * import com.google.cloud.language.v1.Document;
+   * import com.google.cloud.language.v1.LanguageServiceClient;
+   */
+
+  /**
+   * Classifying Content in text file stored in Cloud Storage
+   *
+   * @param gcsContentUri Google Cloud Storage URI where the file content is located. e.g.
+   *     gs://[Your Bucket]/[Path to File] The text file must include at least 20 words.
+   */
+  public static void sampleClassifyText(String gcsContentUri) {
+    try (LanguageServiceClient languageServiceClient = LanguageServiceClient.create()) {
+      // gcsContentUri = "gs://cloud-samples-data/language/classify-entertainment.txt";
+
+      // Available types: PLAIN_TEXT, HTML
+      Document.Type type = Document.Type.PLAIN_TEXT;
+
+      // Optional. If not specified, the language is automatically detected.
+      // For list of supported languages:
+      // https://cloud.google.com/natural-language/docs/languages
+      String language = "en";
+      Document document =
+          Document.newBuilder()
+              .setGcsContentUri(gcsContentUri)
+              .setType(type)
+              .setLanguage(language)
+              .build();
+      ClassifyTextRequest request = ClassifyTextRequest.newBuilder().setDocument(document).build();
+      ClassifyTextResponse response = languageServiceClient.classifyText(request);
+      // Loop through classified categories returned from the API
+      for (ClassificationCategory category : response.getCategoriesList()) {
+        // Get the name of the category representing the document.
+        // See the predefined taxonomy of categories:
+        // https://cloud.google.com/natural-language/docs/categories
+        System.out.printf("Category name: %s\n", category.getName());
+        // Get the confidence. Number representing how certain the classifier
+        // is that this category represents the provided text.
+        System.out.printf("Confidence: %s\n", category.getConfidence());
+      }
+    } catch (Exception exception) {
```

Comment:
This exception is way too broad - we should only be catching exceptions that we have to. For those exceptions, we should be showing the user how to correctly handle them. 

@lesv is working on a proposal to clarify which errors/exceptions we should handle and what to do with them.